pom.xml

```diff
@@ -303,5 +303,10 @@
             <version>2.8.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-configuration</groupId>
+            <artifactId>commons-configuration</artifactId>
+            <version>1.6</version>
+        </dependency>
```

Comment:
also I wonder if you really need this as JDK supports reading from properties file into the java.util.Properties

Reply:
Its more powerful as it maintains load order...using java.util.properties doesn't as it extends a Hashtable...i'd like the order maintained if possible...its up to the person using the library to provide the properties file in whatever order they like....the module maintains that...hope that makes sense?

Reply:
makes sense!